### PR TITLE
add ddl

### PR DESCRIPTION
**Pull request by daviszhen**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

issue #2421 

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

File: pkg/sql/plan2/build_ddl.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func buildCreateTable(stmt *tree.CreateTable, ctx CompilerContext) (*Plan, error
 
 	// get database name
 	if len(stmt.Table.SchemaName) == 0 {
-		createTable.Database = ctx.DefaultDatabase()
+		createTable.Database = ""
```
Comment (ouyuanning) [inline pkg/sql/plan2/build_ddl.go:38]: maybe you can change to :

createTable.database = string(stmt.Table.SchemaName)

if we don't want to get default database.

Comment (iamlinjunhong) [reply pkg/sql/plan2/build_ddl.go:38]: It will change back when DefaultDatabase() return right. Now, DefaultDatabase() will return "mo_catalog"

**Review by iamlinjunhong — APPROVED** [review]

**Review by ouyuanning — APPROVED** [review]

File: pkg/sql/plan2/build_ddl.go (hunk 1 above)
Comment (fengttt) [reply pkg/sql/plan2/build_ddl.go:38]: default database should be current database, that is, the one you connected to (use database).

Comment (fengttt) [reply pkg/sql/plan2/build_ddl.go:38]: @iamlinjunhong Are you saying DafaultDatabase() has a bug or NYI, and we will revert to ctx.DefaultDatabase() once we fix DefaultDatabase?    At least deserve a HUGE XXX comment.

**Pull request merged**